.evergreen/config.yml

```
@@ -291,16 +291,12 @@ functions:
       params:
         # Purposely use sh here to ensure backwards compatibility.
         binary: sh
-        env:
-          MONGODB_VERSION: ${VERSION}
-          TOPOLOGY: ${TOPOLOGY}
-          SSL: ${SSL}
-          AUTH: ${AUTH}
-          STORAGE_ENGINE: ${STORAGE_ENGINE}
-          LOAD_BALANCER: ${LOAD_BALANCER}
-          REQUIRE_API_VERSION: ${REQUIRE_API_VERSION}
+        include_expansions_in_env: [
+          MONGODB_VERSION,TOPOLOGY,SSL,AUTH,STORAGE_ENGINE,LOAD_BALANCER,REQUIRE_API_VERSION,
+          LOCAL_ATLAS]
```

```suggestion
        env:
          MONGODB_VERSION: ${VERSION}
        include_expansions_in_env:
          - AUTH
          - LOAD_BALANCER
          - LOCAL_ATLAS
          - REQUIRE_API_VERSION,
          - SSL
          - STORAGE_ENGINE
          - TOPOLOGY
```
`MONGODB_VERSION` != `VERSION`.